fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -2285,8 +2285,12 @@ setQuantifier
 
 selectItem
     : expression (AS? (identifier | string))?                                            #selectSingle
-    | qualifiedName '.' ASTERISK_SYMBOL                                                  #selectAll
-    | ASTERISK_SYMBOL                                                                    #selectAll
+    | qualifiedName '.' ASTERISK_SYMBOL excludeClause?                                   #selectAll
+    | ASTERISK_SYMBOL excludeClause?                                                     #selectAll
+    ;
+
+excludeClause
+    : EXCLUDE '(' identifier (',' identifier)* ')'
```

Comment:
EXCEPT can be an alternative.
like `( EXCEPT | EXCLUDE )`

Reply:
done